appinventor/appengine/src/com/google/appinventor/client/wizards/RequestNewProjectNameWizard.java

```diff
@@ -46,7 +46,7 @@ public RequestNewProjectNameWizard(RequestProjectNewNameInterface newName,
     final DialogBox db = new DialogBox(false, true);              
     db.setText(title);                                                   // title of the dialog box
     db.setStyleName("ode-DialogBox");
-    db.setAnimationEnabled(true);
+    // db.setAnimationEnabled(true);
```

Comment:
1. What is the reason for this change?
2. If you're going to disable code, it is preferred to remove it rather than comment it out.

Reply:
1.When the dialog box is closing the animation doesn't look smooth compared to the animation when dialog box opens.
After seeing other dialog boxes of the ai2 (like the one in create new project) , I thought it is better to disable the animation like other dialog boxes. 
2.I will remove the commented code.